mule-maven-plugin/src/main/java/org/mule/tools/maven/mojo/PackageMojo.java

```diff
@@ -35,6 +40,53 @@
   @Component
   protected MavenProjectHelper helper;
 
+  /**
+   * Whether a JAR file will be created for the classes in the app. Using this optional configuration parameter will make the
+   * generated classes to be archived into a jar file and the classes directory will then be excluded from the app.
+   *
+   * @parameter expression="${archiveClasses}" default-value="false"
+   */
+  private boolean archiveClasses;
+
+
+  /**
+   * List of exclusion elements (having groupId and artifactId children) to exclude from the application archive.
+   *
+   * @parameter
+   * @since 2.3.0
+   */
+  private ArrayList<Exclusion> exclusions;
+
+  /**
+   * List of inclusion elements (having groupId and artifactId children) to exclude from the application archive.
+   *
+   * @parameter
+   * @since 2.3.0
+   */
+  private ArrayList<Inclusion> inclusions;
+
+  /**
+   * Exclude all artifacts with Mule groupIds. Default is <code>true</code>.
+   *
+   * @parameter default-value="true"
+   * @since 2.3.0
+   */
+  private boolean excludeMuleDependencies;
+
+  /**
+   * @parameter default-value="false"
+   * @since 2.3.0
+   */
+  private boolean filterAppDirectory;
+
+  /**
+   * @parameter default-value="false"
```

Comment:
please all @parameter outside the comment section

Reply:
Idem